src/slg/scene/parsematerials.cpp

```diff
@@ -217,6 +217,7 @@ Material *Scene::CreateMaterial(const u_int defaultMatID, const string &matName,
 			cauchyB = GetTexture(props.Get(Property(propName + ".cauchyb")(0.f, 0.f, 0.f)));
 		// For compatibility with the past
 		else if (props.IsDefined(propName + ".cauchyc"))
```

Comment:
You need to put `{}` around this `else if` scope since it's not a single statement anymore.